functions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * connects to the database and extracts all items required and returns in an associative array
+ * @param $dbName, the name of the db to connect to
+ *
+ * @return array, the array of items in the collection
+ */
+function connect_to_db_and_extract_items($dbName)
+{
+    $db = new PDO('mysql:host=db;dbname=' . $dbName, 'root', 'password'); //initialise the db connection
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare('SELECT `variety`, `tones`, `cost`, `nameOfBrand`, `country`, `wine`.`img_location` FROM `wine` JOIN `brands` ON `wine`.`brand_id` = `brands`.`id` JOIN `country` ON `wine`.`country_of_origin`=`country`.`id`;');
```

Comment:
phpstorm has a vertical line about 80 characters into your screen, this line represents how long a line of code can be before you should wrap it onto several lines. this is one of those times